OpenSim/Simulation/Wrap/WrapEllipsoid.cpp

```diff
@@ -72,62 +69,37 @@ WrapEllipsoid::~WrapEllipsoid()
 {
 }
 
-//_____________________________________________________________________________
-/**
-* Copy constructor.
-*
-* @param aWrapEllipsoid WrapEllipsoid to be copied.
-*/
-WrapEllipsoid::WrapEllipsoid(const WrapEllipsoid& aWrapEllipsoid) :
-   WrapObject(aWrapEllipsoid),
-   _dimensions(_dimensionsProp.getValueDblArray())
-{
-    setNull();
-    setupProperties();
-    copyData(aWrapEllipsoid);
-}
-
 //=============================================================================
 // CONSTRUCTION METHODS
 //=============================================================================
-//_____________________________________________________________________________
-/**
-* Set the data members of this WrapEllipsoid to their null values.
-*/
-void WrapEllipsoid::setNull()
-{
-}
-
 //_____________________________________________________________________________
 /**
 * Connect properties to local pointers.
 */
-void WrapEllipsoid::setupProperties()
+void WrapEllipsoid::constructProperties()
 {
     // BASE CLASS
     //WrapObject::setupProperties();
 
-    const double defaultDimensions[] = {-1.0, -1.0, -1.0};
-    _dimensionsProp.setName("dimensions");
-    _dimensionsProp.setValue(3, defaultDimensions);
-    _propertySet.append(&_dimensionsProp);
+    SimTK::Vec3 defaultDimensions = {1.0, 1.0, 1.0};
```

Comment:
These dimensions of 1.0 meter are huge. Consider using defaults of 1 cm (0.01).

Reply:
1 cm seems a bit tiny, I made them 5 cm to match the outer radius of the torus